src/Features/CSharp/Portable/CodeFixes/HideBase/HideBaseCodeFixProvider.cs

```diff
@@ -17,14 +17,15 @@ namespace Microsoft.CodeAnalysis.CSharp.CodeFixes.HideBase
     internal partial class HideBaseCodeFixProvider : CodeFixProvider
     {
         internal const string CS0108 = nameof(CS0108); // 'SomeClass.SomeMember' hides inherited member 'SomeClass.SomeMember'. Use the new keyword if hiding was intended.
+        internal const string CS0109 = nameof(CS0109); // The member 'SomeClass.SomeMember' does not hide an accessible member. The new keyword is not required.
```

Comment:
ah, i see you reused this type.   i would't just make a new type.

Reply:
@CyrusNajmabadi is that "would" or "wouldn't"?

Reply:
fixed :)

Reply:
Moved to `Microsoft.CodeAnalysis.CSharp.CodeFixes.RemoveNewModifier.RemoveNewModifierCodeFixProvider`.